src/executors/default-with-neo4j-cluster.yml

```diff
@@ -0,0 +1,104 @@
+parameters:
+  golang-image-version:
+    type: string
+    default: "1"
+  neo4j-image-version:
+    type: string
+    default: "4.3.6-enterprise"
+  neo4j-test-url:
+    type: string
+    default: "neo4j://core0:7687"
+docker:
+  - image: golang:<<parameters.golang-image-version>>
+    auth:
+      username: $DOCKERHUB_USERNAME
+      password: $DOCKERHUB_ACCESS_TOKEN
+    environment:
+      NEO4J_TEST_URL: <<parameters.neo4j-test-url>>
+
+  - image: neo4j:<<parameters.neo4j-image-version>>
+    name: core0
+    auth:
+      username: $DOCKERHUB_USERNAME
+      password: $DOCKERHUB_ACCESS_TOKEN
+    environment:
+      NEO4J_AUTH: none
+      NEO4J_HEAP_MEMORY: 256
+      NEO4J_CACHE_MEMORY: 256M
```

Comment:
**thought:** I think that these 2 options are not valid. Maybe we need these? 
- [NEO4J_dbms_memory_heap_initial__size](https://neo4j.com/docs/operations-manual/current/reference/configuration-settings/#config_dbms.memory.heap.initial_size)
- [NEO4J_dbms_memory_heap_max__size](https://neo4j.com/docs/operations-manual/current/reference/configuration-settings/#config_dbms.memory.heap.max_size)
- [NEO4J_dbms_memory_pagecache_size](https://neo4j.com/docs/operations-manual/current/reference/configuration-settings/#config_dbms.memory.pagecache.size)

Reply:
I was trying to figure out why those were added on the first place but I couldn't find out. Looking at the very useful links above, those settings/env vars have very sensible default values. So I decided to remove those settings from both Neo4j executors. The change is tested:
- for the cluster executor check [the latest cmneo4j commit](https://github.com/Financial-Times/cm-neo4j-driver/commit/a3482ec35fd10a92e514e0ef5db747575ff7022e) and [CI run](https://app.circleci.com/pipelines/github/Financial-Times/cm-neo4j-driver/14/workflows/c004684d-046d-4717-95fd-1bbad225f75e/jobs/28);
- for the single Neo executor check [the neo4j-metric-aggregator commit ](https://github.com/Financial-Times/neo4j-metric-aggregator/commit/c5705613b8a7fa6bb24e08ae050f8da0a612ebe5)and [CI run](https://app.circleci.com/pipelines/github/Financial-Times/neo4j-metric-aggregator/83/workflows/12fd5709-8400-424a-8e02-fd1571615918/jobs/147);